contracts/ToucanRegenBridge.sol

```diff
@@ -66,20 +66,22 @@ contract ToucanRegenBridge is Ownable, Pausable, AccessControl {
         uint256 accountLen = account.length;
         require(accountLen == 44 || accountLen == 64, "regen address must be 44 or 64 chars");
 
-        // verification: check address starts with "regen1" prefix
-        bytes memory prefix = "regen1";
-        for (uint8 i = 0; i < 6; ++i)
-            require(prefix[i] == account[i], "regen address must start with 'regen1'");
-
-        // verification: check address contains only alphanumeric characters
-        for (uint64 i = 0; i < accountLen; i++) {
-            bytes1 char = account[i];
-            require(
-                (char >= 0x30 && char <= 0x39) || //9-0
-                    (char >= 0x41 && char <= 0x5A) || //A-Z
-                    (char >= 0x61 && char <= 0x7A), //a-z
-                "regen address must contain only alphanumeric characters"
-            );
+        unchecked {
+            // verification: check address starts with "regen1" prefix
+            bytes memory prefix = "regen1";
+            for (uint8 i = 0; i < 6; ++i)
+                require(prefix[i] == account[i], "regen address must start with 'regen1'");
+
+            // verification: check address contains only alphanumeric characters
+            for (uint8 i = 6; i < accountLen; ++i) {
```

Comment:
nit: isn't it actually cheaper to use `uint256` here and in the loop above instead of `uint8`? It'd be great to run these over [gas-reporter](https://www.npmjs.com/package/hardhat-gas-reporter).

Reply:
I conducted a small tests using remix and it seems that in newer compilers using small sized types are using less gas.  But I am running same test using gas-reporter as well